apps/src/gamelab/GameLabGroup.js

```diff
@@ -76,13 +76,28 @@ module.exports.Group = function (baseConstructor) {
    * @return {boolean} True if any touching occurred
    */
   array.isTouching = function (target) {
-    var didTouch = false;
-    for (var i = 0; i < this.length; i++) {
-      didTouch = this[i].isTouching(target) || didTouch;
-      // Note: can't early out because each call needs to update the sprite's
-      // touching property.
+    var state = jsInterpreter.getCurrentState();
```

Comment:
It would be nice to add a comment to this method's short description indicating that it follows the approach used in _groupCollideGameLab, which has a much longer explanation of how substate and interpreter variables are used.